test/e2e/utils/cluster.go

```diff
@@ -34,11 +36,11 @@ func GetClusterInfo(config configclient.Interface) (*configv1.Infrastructure, er
 }
 
 // GetCredentials searches for AWS or BMH secret, and then returns it decoded
-func GetCredentials(clientSet *kubernetes.Clientset, secretName, secretKey, secretVal string) (string, string, error) {
+func GetCredentials(clientSet *kubernetes.Clientset, secretName, secretNamespace, secretKey, secretVal string) (string, string, error) {
```

Comment:
misleading function name and description: this is not specific to AWS or BMH, it's just getting a secret's key and value based on function parameters